### PR TITLE
Use MutateObserver to prevent loading lag

### PR DESCRIPTION
**Pull request by yxlin118**
issue #19

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,4 +21,4 @@
   "devDependencies": {
     "parcel-bundler": "^1.12.4"
   }
-}
\ No newline at end of file
+}
```
Comment (navendu-pottekkat) [inline package.json:24]: Remove the extra space. This file should not be shown as changed file

File: src/content.js [hunk 2]
```diff
@@ -49,23 +49,18 @@ function analyzeImage(image) {
...
-// Call function when DOM is loaded
-window.addEventListener("load", () => {
-  clasifyImages()
-
-  // @todo hanlde with javascript render delay
-  const timesToJSrun = [500, 1000, 2000, 3000]
-  for (let i = 0; i < timesToJSrun.length; i++) {
-    setTimeout(() => { clasifyImages() }, timesToJSrun[i]);
-  }
+//Call function when changes made in DOM tree
```
Comment (navendu-pottekkat) [inline src/content.js:59]: When you load a web page at first the images are hidden without any lag. But when the same page is reloaded, the filter does not work. Also I am experiencing considerable slowing down of my browser while running the extension that did not happen before

**Comment by navendu-pottekkat** [comment]
@yxlin118 Please test it once again and check if you are experiencing any issues

File: src/content.js (hunk 2 above)
Comment (yxlin118) [reply src/content.js:59]: Which page are you testing with? And when you say reload, does that mean refreshing?(I tried https://scrolller.com/nsfw) and it seems to work. Is there a way to see the slowing down of the browser? I didn't notice it.

**Comment by YegorZaremba** [comment]
@yxlin118 Thanks for contributing, pls check my PR #26, now we can handle dynamic JS rendering, now if you'll go to website in README and choose in settings only `pics`, we detect new image with `MutationObserver`

But your bug #19 is still exists, I mean Im not sure thats about dynamic render or not, it takes <250ms to hide every image, but I still have no idea how to remove this time lag, hope we will find solution

File: src/content.js (hunk 2 above)
Comment (navendu-pottekkat) [reply src/content.js:59]: I tested with Google Images. I tried to reload/refresh on the browser and the filter did not seem to work then. I tried with the current MASTER branch and the PR and there was a considerable lag between them. Could you test it out thoroughly?

**Comment by YegorZaremba** [comment]
v1.0.0 is merged #29, thanks for your contribution

**Pull request closed without merging**